website/content/docs/connect/gateways/api-gateway/configuration/gatewaypolicy.mdx

```diff
@@ -0,0 +1,257 @@
+---
+layout: docs
+page_title: GatewayPolicy configuration reference
+description: Learn about the configuration options for the GatewayPolicy configuration resource. GatewayPolicy resources define API gateway polcies for Consul service mesh on Kubernetes.
+---
+
+# GatewayPolicy
+
+This topic provides reference information for the API gateway policy resource, which defines API gateway policies for Consul service mesh on Kubernetes-orchestrated networks.
+
+## Configuration model
+
+The following list outlines field hierarchy, data types, and requirements in a gateway policy resource. Click on a property name to view additional details, including default values.
+
+- [`apiVersion`](#apiversion): string | required | must be set to `consul.hashicorp.com/v1alpha1`
+- [`kind`](#kind): string | required | must be set to `GatewayPolicy`
+- [`metadata`](#metadata): map | required
+   - [`name`](#metadata-name): string | required 
+   - [`namespace`](#metadata-namespace): string  | `default`
+- [`spec`](#spec): map | required
+   - [`targetRef`](#spec-targetref): map | required 
+      - [`name`](#spec-targetref): string | required 
+      - [`kind`](#spec-targetref): string | required | must be set to `Gateway`
+      - [`group`](#spec-targetref): string | required | must be set to `gateway.networking.kuberenetes.io`
```

Comment:
```suggestion
      - [`group`](#spec-targetref): string | required 
```
If it can be a different resource group, then we shouldn't say that any specific string is required.